packages/babel-plugin-strip-runtime/src/types.ts

```diff
@@ -28,26 +38,19 @@ export interface PluginPass {
   /**
    * Data of the current file being transformed.
    */
-  file: {
-    opts: {
-      generatorOpts?: {
-        sourceFileName?: string;
-      };
-    };
-    metadata: {
-      styleRules: string[];
-    };
-  };
+  file: BabelFile;
 
   /**
-   * Current filename
+   * The name of the jsx function specified in the JSX pragma
+   * "@ jsx nameOfJsxFunction". (Classic syntax)
    */
-  filename: string;
```

Comment:
filename and cwd still exist, they're in `OriginalPluginPass` now

Reply:
https://github.com/babel/babel/blob/master/packages/babel-core/src/transformation/plugin-pass.js